action.yml

```diff
@@ -8,7 +8,7 @@ inputs:
     description: >
       The branch, tag or SHA to checkout. When checking out the repository that
       triggered a workflow, this defaults to the reference or SHA for that
-      event.  Otherwise, defaults to `master`.
+      event.  Otherwise, uses the default branch.
   token:

```

Comment:
Could we update this to have `required: true`?

Reply:
It's not required that the user enter a value. It has a default.